src/lib/object-helpers.js

```diff
@@ -0,0 +1,88 @@
+'use strict'
+
+/**
+ * Use to break arrays up into 'chunks'
+ * @module Chunk
+ */
```

Comment:
Just a copy & paste thing to correct! 😁 

```suggestion
/**
 * Collection of helper functions that allowed us to move away from Lodash
 * @module ObjectHelpers
 */
```